### DISCUSSION TITLE
how to delete issues?

### DISCUSSION THREAD

**Discussion by thilllon** [octokit/octokit.js#2343] [Q&A] [answered]
I can't find `Delete an issue` API from the docs. Please let me know how to delete issues with octokit.

https://octokit.github.io/rest.js/v19#issues-create
https://docs.github.com/en/rest/issues/issues

**Comment by kfcampbell** [comment, marked as answer]
You can update the status to "closed" using the issue update API, but you're correct that there's no API to permanently delete an issue and thus Octokit is unable to support it. I don't believe the functionality is exposed through GraphQL either.

**Reply by gr2m** [reply]
You can ask for a new API to be created at https://github.com/community/community/discussions/categories/api-and-webhooks
